### PR TITLE
Create command to update a field of a user with a given value

### PR DESCRIPTION
**Pull request by Advayp**
## Description

A simple command to update the value of any given and allowed field with an appropriate value.

### CODE REVIEW

File: server/members/management/commands/update_field.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                value = int(value)
+
+            user_to_update.__setattr__(field, value)
+
+            user_to_update.full_clean()
+            user_to_update.save()
+
+            self.stdout.write(
+                self.style.SUCCESS(
+                    f"Successfully updated field {Command.output_info(field)} with value {Command.output_info(value)} for user {Command.output_info(username)}."
```
Comment (mjh316) [inline server/members/management/commands/update_field.py:90]: I think that if `value` is an int, output_info(value) will throw a type error here as it attempts to concatenate an integer and strings. Could you let me know if that's the case?

File: server/members/management/commands/update_field.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            self.stdout.write(
+                self.style.ERROR(
+                    f"Cannot write to type {Command.output_info(field_type)}."
+                )
+            )
+            return
+
+        try:
+            if field_type == models.BigIntegerField.__name__:
+                value = int(value)
```
Comment (mjh316) [inline server/members/management/commands/update_field.py:81]: The line that might cause an issue for what I'm referencing in the comment below

**Review by mjh316 — CHANGES_REQUESTED** [review]

File: server/members/management/commands/update_field.py (hunk 1 above)
Comment (Advayp) [reply server/members/management/commands/update_field.py:90]: Yup, you're right. I forgot python doesn't automatically cast to string. I'll fix once I'm back home.

File: server/members/management/commands/update_field.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        return "`" + str(field_name) + "`"
+
+    help = f"Command to set a particular field of a user to a given value. Disallowed fields include: {', '.join(map(output_info, disallowed_fields))}."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--username", type=str, help="SWECC Interview Website Username"
+        )
+        parser.add_argument(
+            "--field", type=str, help="Field to update", default="username"
```
Comment (mjh316) [inline server/members/management/commands/update_field.py:38]: Looks fine, do we want to say anywhere that username is the default?

**Review by mjh316 — APPROVED** [review]

**Pull request merged**